cpp/FileTransferAgent.hpp

```diff
@@ -170,6 +172,25 @@ class FileTransferAgent : public IFileTransferAgent
     m_useDevUrand = useUrand;
   }
 
+  virtual void setPutFastFail(bool fastFail)
+  {
+    if(fastFail)
+    {
+      CXX_LOG_DEBUG("Enabling put fast fail.");
+    }
+    m_fastFail = fastFail;
```

Comment:
Shouldn't `m_fastFail = fastFail` be inside the if statement? Or do you even need this if statement at all?

Reply:
The function intention is to set the value of fastFail into m_fastFail which gets done irrespective of the if condition. 
And the log message is for debugging purpose.